src/org/opensolaris/opengrok/web/ResponseHeaderFilter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The contents of this file are Copyright © 2003 Jayson Falkner made available
+ * under free license: "All of the code from Servlets and JSP the J2EE Web Tier
+ * is freely available. Additonally all of the code the book's code relies on
+ * is also freely available. With the exception of Sun's Java Development Kit,
+ * everything the book relies on is also open-source. You are encouraged to
+ * look at, learn from, and use everything!"
+ */
+
+package org.opensolaris.opengrok.web;
+
+import java.io.*;
```

Comment:
pls use specific imports, in OpenGrok code base we try to avoid wildcard imports.